generators/server/templates/src/main/java/package/service/mapper/_UserMapper.java

```diff
@@ -54,7 +53,7 @@ default User userFromId(<% if (databaseType == 'mongodb' || databaseType == 'cas
     }
 
     default Set<Authority> authoritiesFromStrings(Set<String> strings) {
-        return strings.stream().map(string -> {
+        return strings == null ? null : strings.stream().map(string -> {
```

Comment:
Not sure about this: if the front end does send a null value, why don't we have an NPE? Anyway I don't like to have null values hanging around